docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -0,0 +1,518 @@
+---
+title: "Migrate from Newtonsoft Json.NET to System.Text.Json - .NET"
+author: tdykstra
+ms.author: tdykstra
+ms.date: "12/02/2019"
+helpviewer_keywords: 
+  - "JSON serialization"
+  - "serializing objects"
+  - "serialization"
+  - "objects, serializing"
+---
+
+# How to migrate from Newtonsoft Json.NET to System.Text.Json
+
+This article shows how to migrate from [Newtonsoft Json.NET](https://www.newtonsoft.com/json) to the JSON serialization tools in the [System.Text.Json namespace](system-text-json-overview.md).
+
+`System.Text.Json` is relatively new, and its initial focus is on performance, security, and standards compliance. It lacks built-in features to handle some common scenarios, and some default behaviors differ from Newtonsoft. For some scenarios, `System.Text.Json` has no built-in functionality, but therre are recommended workarounds. For other scenarios,  workarounds are impractical. If your application depends on a missing feature that has no workaround, consider delaying your migration to `System.Text.Json`. 
```

Comment:
I would remove the time-based clause here: "is relatively new" as something is only new for a short period of time. I would also refrain from saying "Initially" as it brings in the idea that there is a guarantee that this will change. But we have no guarantee that the API evolves beyond anything it currently does. I would just state:

>`System.Text.Json` currently focuses on performance, security, and standards compliant. Compared to Newtonsoft JSON, it lacks X Y Z. .... etc

Also, some typos in the current text, a double space and "therre"

```suggestion
`System.Text.Json` is relatively new, and its initial focus is on performance, security, and standards compliance. It lacks built-in features to handle some common scenarios, and some default behaviors differ from Newtonsoft. For some scenarios, `System.Text.Json` has no built-in functionality, but there are recommended workarounds. For other scenarios, workarounds are impractical. If your application depends on a missing feature that has no workaround, consider delaying your migration to `System.Text.Json`. 
```